prow/cluster/jobs/istio/operator/istio.operator.yaml

```diff
@@ -2,6 +2,45 @@ presubmits:
 
   istio/operator:
 
+  - name: operator-lint-dind
+    path_alias: istio.io/operator
+    decorate: true
+    always_run: true
+    optional: true
+    spec:
+      containers:
+      - image: docker.io/sdake/prow-launcher:2019-08-03
+        command:
+        - entrypoint
+        - make
+        - lint
+        resources:
+          limits:
+            cpu: "5"
+            memory: 24Gi
+          requests:
+            cpu: "3"
+            memory: 3Gi
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /lib/modules
```

Comment:
do we need this for dind? I thought it was just kind is special somehow. afaik we have other tests using docker (hadolint runs in docker?) without this.

Also why do we need dind for CI? can't we just set the image to our new build container

Reply:
this is prototype work.  We can't just set the image to our new build container because we want to run make commands naturally without non-CI workflows in the makefiles.   See this PR: https://github.com/istio/operator/pull/151

Notice make lint fails, but the others succeed because they are callling `go build` and `go test` directly.

FWIW @geeknoid has greenlit this work :)

Reply:
Not sure on the volume mounts, I just want something that works for the moment and copied the install - as it uses a complex Kind in K8s model.  Can refine after prototype work is done.